_posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md

```diff
@@ -0,0 +1,100 @@
+
+Build a decentralized exchange either with an automated market maker or a match-making mechanism where trades do not really work as expected. This can be either by it leaking money, a specific account being able to withdraw all money or something else you can think of. You can also add a flaw to a token implementation you provide instead of to the exchange itself. The only hard requirement is that the flaw is hidden.
+
+## The Judges ⚖️
+
+Judges are presented with anonymised submissions. This year, the submissions will be assessed by:
+
++ [Alex Beregszaszi](https://twitter.com/alexberegszaszi), Solidity Co-Lead at Ethereum Foundation.
++ [Anton Permenev](https://twitter.com/a_permenev), Senior Engineer at ChainSecurity.
++ [Duncan Townsend](https://twitter.com/duncancmt), CTO at Immunefi.
++ [Gonçalo Sá](https://twitter.com/GNSPS), Security Engineer at ConsenSys Diligence.
++ [Harikrishnan Mulackal](https://twitter.com/_hrkrshnn), C++ Engineer Solidity at Ethereum Foundation.
++ [Josselin Feist](https://twitter.com/Montyly), Principal Security Engineer at Trail of Bits.
++ [samczsun](https://twitter.com/samczsun), Research Partner at Paradigm.
++ [Stefan Beyer](https://twitter.com/beyer_st), Lead Auditor at Solidified.
+
+### Judging Criteria ☑️
+
+Rember to consider the following aspects and properties for your submissions:
+
++ **Simplicity & Briefness**: Since it’s much easier to hide a vulnerability in complex and poorly written code, short and clean submissions will be scored higher than those that are lengthy and complicated.
++ **Plausibility**: Code that drops down to inline assembly without any clear reason why will look immediately suspicious, no matter how cleverly written the assembly-level flaw is.
++ **Originality**: Truly original ideas will receive more points than making use of already well known exploit/backdoor mechanisms.
+
+## The Prizes 🏆
+
+This year, the first place will receive a ticket to [Devcon VII Bogota](https://devcon.org/).
+
+The top 3 submissions will receive a ticket to the [Devconnect Coworking Space](https://devconnect.org/) and the opportunity to present their contest submission at the [Solidity Summit 2022](https://summit.soliditylang.org/) in Amsterdam.
+
+The top 3 submissions of the Underhanded Solidity Contest will also be awarded points for the upcoming [Paradigm CTF 2022](https://ctf.paradigm.xyz/).
+Those points will be awarded as though the winners solved a bonus challenge ("pseudo-challenge").
+
+Furthermore, the three winners will be added to the Board of Fame.
+
+The winners and all qualified submissions will receive a custom Underhanded Solidity Contest NFT.
+
+## The Submission 📨
+
+Before you submitting your code ensure that:
++ The entirety of your submission is licensed under an open-source license.
++ In your submission you use Solidity v0.8.0 or newer.
++ You have carefully read the [Coding Brief](https://underhanded.soliditylang.org/#coding-brief).
+
```

Comment:
Do we want to re-state that it should contain a readme and a separate spoiler / rugpull.txt?

Reply:
done